src/Devices/PushingBox.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include <Arduino.h>
+
+#include "Ethernet_TC.h"
+
+class PushingBox {
+public:
+  // class methods
+  static PushingBox *instance();
+
+  // instance methods
+  EthernetClient *getClient() {
+    return &client;
+  }
+  byte *getServer() {
+    return server;
+  }
+  void loop();
+
+private:
+  // class variables
+  static PushingBox *_instance;
+
+  // instance variables
+  EthernetClient client;
+  const char *DevID = "v172D35C152EDA6C";  // DeviceID from Pushingbox
+  unsigned long nextSendTime = 0;
+  byte server[4] = {64, 233, 187, 99};
```

Comment:
In production the server should be `api.pushingbox.com`. We should think about how the mock server will handle that...